library/user/reconnaissance/live_walk/payload.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Title: live_walk
+# Description: A WiFi logger that records networks in detail as you walk or perform audits . 
+# Author: MusicalVR
+
+LOOTDIR="/root/loot/live_walk"
+mkdir -p "$LOOTDIR"
+IFACE="wlan1mon"
+PREFIX="walk_$(date +%H%M)"
+FULL_PATH="$LOOTDIR/$PREFIX-01.csv"
+
+# Log systems
+cleanup() {
+    LOG "!!!"
+    LOG "Stopping Capture..."
+    kill "$PID" 2>/dev/null
+    killall airodump-ng > /dev/null 2>&1
+    LOG "Saved: $PREFIX"
+    exit 0
+}
+
+# Startup
+killall -9 airodump-ng > /dev/null 2>&1
+killall -9 pineapd > /dev/null 2>&1
```

Comment:
This should use `service pineapd stop` to properly stop the pineapd service, and the clean up / exit function (which should trap on exit) should start the service again with `service pineapd start`